chain/ethereum/ethereum.go

```diff
@@ -163,14 +159,23 @@ func ChunkedFilterLogs(ctx context.Context, client ETHClient, addresses []common
 			return nil, err
 		}
 		log.Warn().Err(err).Uint64("from", fromBlock).Uint64("to", toBlock).Msg("skipping failed backfill interval...")
-		failedQueries = append(failedQueries, query)
+
+		mid := (query.FromBlock.Uint64() + query.ToBlock.Uint64()) / 2
```

Comment:
why do we split in half? not requesting changes, just curious

Reply:
sometimes node providers think the request is too large. It could be block range, number of events or just response size. They refuse the call.
I wanted to divide these calls so they are smaller for the provider and can go through.
I will add this as a comment too